Sprint-1/explore/chrome.md

```diff
@@ -9,10 +9,16 @@ Let's try an example.
 
 In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
+* A popup apprears saying "Hello world! As well as a confirmation "OK" buttom
 
 What effect does calling the `alert` function have?
+* It gives you an alert which will remain there until you acknowledge it by clicking ok.
+
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
 What effect does calling the `prompt` function have?
+* * A popup apprears with the question, an input field, and an ok and cancel button.
+
 What is the return value of `prompt`?
+* The return value is undefined. However, the name I added to the input is saved in the variable myName
```

Comment:
`prompt()` should never return `undefined`. If the user clicked "Cancel", `prompt()` would return `null` (which is not equal to `undefined` in JavaScript).